web/src/components/Inputs.js

```diff
@@ -2,17 +2,25 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { Box, Input, Label, Textarea } from 'rebass';
 
+const offscreenStyle = {
```

Comment:
hiding labels (and keeping them still visible for screen readers) is perfectly legit, and your CSS to do this looks good. 18F's accessibility guide offers a slightly modified CSS snippet to accomplish the same thing (https://accessibility.18f.gov/hidden-content/) but they admit "There are multiple ways to accomplish this"